Makefile

```diff
@@ -81,9 +81,12 @@ bin/k8s:
 bin/unpack:
 	CGO_ENABLED=0 go build -o $@ ./cmd/unpack/...
 
-build-local-container: ## Builds provisioner container image locally
+build-container: ## Builds provisioner container image locally
 	docker build -f Dockerfile -t $(IMAGE) .
 
+build-local-container: build
```

Comment:
could we add a Make comment here to explain the target? Something simple like
`## Builds the provisioner container image using locally built binaries` sounds fine.

Reply:
I'll add it.  Thanks.